### ISSUE TITLE
Search for coordinates does not work

### ISSUE THREAD

**Issue by googie0815** [umap-project/umap#1543]
**Describe the bug**
If I go to Search location, it says "type a place name or coordinates". If I enter the coordinates, nothing happens. In which format do these have to be entered? I have tried the following format: "48.62963216474561, 7.6849514087986055"
Even shorter does not work (two to five decimal places). #1480 

**To Reproduce**
1. Open the card of your choice
2. Click on the magnifying glass on the left
3. Enter the coordinates in the menu that opens on the right
4. press Enter on the keyboard
5. nothing happens

**Expected behavior**
The map section should jump to the corresponding point. Ideally, the corresponding point should be highlighted with a pin.

**Screenshots**
![Unbenannt1](https://github.com/umap-project/umap/assets/56229424/155eb095-af9b-413d-8953-98f88c6ff5c5)
![Unbenannt2](https://github.com/umap-project/umap/assets/56229424/2d1b319e-df18-4abb-9d15-884dacaf9a62)

**Desktop (please complete the following information):**
 - OS: Win10
 - Browser: Firefox 120.0.1

**Comment by yohanboniface** [comment]
Yes, sorry, there is a misconfiguration, this will be fixed in next release! see [#1000](https://github.com/umap-project/umap/issues/1000#issuecomment-1892220488) for context
